portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,298 @@
+            </div>
+          </div>
+        </div>
+      </section>
+      <hr class="m-0" />
+
+      <!-- Skills-->
+      <section class="resume-section" id="skills">
+        <div class="resume-section-content">
+          <h2 class="mb-5">Skills</h2>
+          <div class="subheading mb-3">Programming Languages & Tools</div>
+          <ul class="list-inline dev-icons">
+            <li class="list-inline-item"><i class="fab fa-java"></i></li>
+            <li class="list-inline-item"><i class="fab fa-python"></i></li>
+            <li class="list-inline-item"><i class="fab fa-html5"></i></li>
+            <li class="list-inline-item"><i class="fab fa-css3-alt"></i></li>
+            <li class="list-inline-item"><i class="fab fa-js-square"></i></li>
+            <li class="list-inline-item"><i class="fab fa-react"></i></li>
+            <li class="list-inline-item"><i class="fab fa-unity"></i></li>
+          </ul>
+        </div>
+      </section>
+      <hr class="m-0" />
+      <!-- Interests-->
+      <section class="resume-section" id="interests">
+        <div class="resume-section-content">
+          <h2 class="mb-5">Interests</h2>
+          <p>
+            Apart from Computer Science, I really enjoy dancing. I picked up the
+            Indian classical dance- Bharatanatyam- when I joined NUS. It allows
+            me to stay connected to my roots. 
+            Dancing is like meditation to me as it requires intense focus and discipline. 
+            It may be surprising to know how logical and mathematical a dance form can be - not too far off from CS!
+          </p>
+          <p class="mb-0">
+            Recently, I picked up sketching portraits. I love drawing portraits
+            of famous actors! As I am still a beginner, I redraw sketches done
+            by other artists- copying one stroke at a time! Hopefully, one day I
+            don't need to refer to other's drawings :") Here are some of my
+            latest drawings!
+          </p>
+
+          <div class="image-gallery">
+            <div class="image-gallery-item">
+              <img
+                class="img-fluid rounded float-left gallery-img"
+                src="assets/img/docStrange"
+                alt=""
+              />
+            </div>
+            <div class="image-gallery-item">
+              <img
+                class="img-fluid rounded float-left gallery-img"
+                src="assets/img/devsena"
+                alt=""
+              />
+            </div>
+            <div class="image-gallery-item">
+              <img
+                class="img-fluid rounded float-left gallery-img"
+                src="assets/img/professor"
+                alt=""
+              />
+            </div>
+          </ul>
+        </div>
+      </section>
+      <hr class="m-0" />
     </div>
+    <!-- Bootstrap core JS-->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
good job on the script tag optimization!
you may want to look for async and defer property of script tag as well, however as it is not supported by all browsers yet, it is fine to do this for now